papers/baremodels.py

```diff
@@ -601,6 +605,55 @@ def json(self):
             'classification': self.oa_status,
             })
 
+    def bibtex(self):
+        """
+        BibTeX representation of the paper, for citation purposes
+        """
+        entry = {
+            'ENTRYTYPE': 'article',
```

Comment:
Bibtex entry type should probably depend on the `doctype` of the paper. I think we should have a mapping between Dissemin's doctypes and the bibtex equivalents through the ORCID import of bibtex, but I did not yet look into this much.